client/App/pages/LandingPage/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import styles from './Dropdown.module.scss';
+import hand from '../../../../assets/hand.svg';
+import home from '../../../../assets/home.svg';
+import leaf from '../../../../assets/leaf.svg';
+import business from '../../../../assets/business.svg';
+import party from '../../../../assets/party.svg';
+import repair from '../../../../assets/repair.svg';
+
+export const Dropdown = () => {
+  return (
+    <div className={styles.dropdown}>
+      <ul className={styles.dropdown__list}>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={hand} alt="hand" />
+          <a>Dla Ukrainy</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={leaf} alt="leaf" />
+          <a>Ogród</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={business} alt="business" />
+          <a>Usługi dla biznesu</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={home} alt="home" />
+          <a>Budowa domu</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={party} alt="party" />
+          <a>Organizacja imprez</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <img className={styles.icon} src={repair} alt="repair" />
+          <a>Montaż i naprawa</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <a>Elektryk</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <a>Projektowanie</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <a>Usługi finansowe</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <a>Hydraulik</a>
+        </li>
+        <li className={styles.dropdown__item}>
+          <a>Remont</a>
+        </li>
+        <li className={styles.dropdown__item}>
```

Comment:
Type script in my editor doesn't like this bem notation. I think that in our project we don't need bem, and we can use classes in  🐪  case notation :) 
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/29579798/166300748-18db308f-18f7-439d-83b2-590e8ea1fc59.png">